english/staypuft/place-hat/place-hat.md

```diff
@@ -1 +1,7 @@
-The marshmallow man springs to life and exclaims "Happy Birthday!"
\ No newline at end of file
+The marshmallow man springs to life and exclaims "Happy Birthday!"
+
+"How did you know", you reply.
+
+["I am your father's brother's nephew's cousin's former roommate." replied the snowman.](how-did-you-know/roommate.md)
```

Comment:
It looks like this link is broken.  You can view the page on GitHub on your fork and click the link there to check if it is working correctly.